client/datatypes.go

```diff
@@ -14,6 +14,12 @@ package client
 
 import "net/http"
 
+// Field user type.
+type Field struct {
```

Comment:
Maybe this place is a nice inspiration of what others think what makes up a field: https://secure.phabricator.com/book/phabricator/article/custom_fields/#custom-field-configurati